client/components/navbar.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Navbar, Form, FormControl, Button } from 'react-bootstrap';
+
+class Nav extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = ({
+      input: '',
+      data1: [],
+      data2: []
```

Comment:
Ok, for now, you should just make a small adjustment. `data1` and `data2` are pretty awful names, since they are actually a `wallet` and `transactions`.